package.json

```diff
@@ -78,6 +78,8 @@
         "core-js": "^3.0.1",
         "d3-hierarchy": "^1.1.4",
         "d3-interpolate": "^1.1.4",
+        "d3-sankey": "0.7.1",
+        "d3-sankey0.12.3": "npm:d3-sankey@0.12.3",
```

Comment:
Wjat os tjos?

Reply:
The latest version of the Sankey library looks better but breaks IE, so we're using different versions depending on the Browser.